model-optimizer/extensions/ops/merge.py

```diff
@@ -34,13 +34,15 @@ def merge_infer(node: Node):
 
             inferred_and_executable = [n for n in node.in_nodes().values() if n['is_partial_inferred'] and
                                        'executable' in n and n['executable']]
-            tensor = inferred_and_executable[0]
-
-            if all([tensor.has_valid('value') and n.has_valid('value') and strict_compare_tensors(tensor.value, n.value)
-                    for n in inferred_and_executable]):
-                node.out_node().value = tensor.value.copy()
-            else:
-                node.out_node().value = None
+            if len(inferred_and_executable) > 0:
```

Comment:
please add a comment about why this is acceptable case when there is no executable branches for `Merge` node as we agreed on the meeting.

```
In case of nested conditional flow in a graph, Merge node can accept no executable branches
```